### PR TITLE
Add .editorconfig & moban sync

### LINKED ISSUE #331: Add .editorconfig file
Our style follows the example given on http://editorconfig.org/ for Python and YAML
label: difficulty/low initiatives/gci
Opened by @mohitkyadav at [Gitter](https://gitter.im//mohitkyadav?at=5a07240b026aeaa02fffbf13)

### PR DESCRIPTION
**Pull request by Kiy4h**
Closes #331

# Reviewers Checklist

- [x] Appropriate logging is done.
- [x] Appropriate error responses.
- [x] Handle every possible exception.
- [x] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [x] See that 100% coverage is there.
- [x] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -18,6 +18,7 @@ addopts =
   -p no:logging
   --color=yes
   --doctest-glob=*.rst
+  --doctest-modules
```
Comment (nvzard) [inline setup.cfg]: @Kiy4h you'll have to delete this line for the builds to pass. A pull request has already been made to solve this issue https://github.com/timofurrer/ramlient/pull/62. For not just undo this change.

**Comment by jayvdb** [comment]
CI is red.  The message says something is wrong with your commit.

**Comment by jayvdb** [comment]
ack https://github.com/coala/corobo/pull/644/commits/1de93b633a9f351649a47c62ee3af0fa1da873e9

**Comment by jayvdb** [comment]
@gitmate-bot ff
